paddle/operators/math/math_function.cu

```diff
@@ -273,6 +273,33 @@ void set_constant_with_place<platform::CUDAPlace>(
                            TensorSetConstantGPU(context, tensor, value));
 }
 
+template <typename T>
+__global__ void RowwiseAddKernel(const T* a, const T* b, T* c, int64_t height,
+                                 int64_t width) {
+  int64_t num = height * width;
+  for (int i = blockIdx.x * blockDim.x + threadIdx.x; i < num;
+       i += blockDim.x * gridDim.x) {
+    int h = i / width;
+    int w = i % width;
```

Comment:
The integer modulo (%) and division operations are expensive in GPU hardware. 
The division operation seems can be replaced by the multiplication. And modulo (%) operations can be replaced by subtraction and multiplication.

Reply:
Done. Thank you!